custom-fonts.md

```diff
@@ -23,6 +41,72 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+### Using the font
+
+Second use the font by delcaring the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html)
+as a constant. 
+Then set the [`TextStyle`](https://docs.flutter.io/flutter/painting/TextStyle-class.html) `fontFamily` property 
+and use the font family specified in the `pubsec.yaml` delcarations. 
+The font family `pubsec.yaml` declaration will look something like `family: [declared-font-family]`.
```

Comment:
I don't understand what you're after here. I think I would delete the last sentence, and think there is enough context and an added example to deduce the context to get the job done here. Since you're seeing something I'm not at the moment, could you provide what you think you'd like to see?